darkflow/net/yolov2/train.py

```diff
@@ -56,19 +53,20 @@ def loss(self, net_out):
     net_out_reshape = tf.reshape(net_out, [-1, H, W, B, (4 + 1 + C)])
     coords = net_out_reshape[:, :, :, :, :4]
     coords = tf.reshape(coords, [-1, H*W, B, 4])
-    adjusted_coords_xy = expit_tensor(coords[:,:,:,0:2])
-    adjusted_coords_wh = tf.sqrt(tf.exp(coords[:,:,:,2:4]) * np.reshape(anchors, [1, 1, B, 2]) / np.reshape([W, H], [1, 1, 1, 2]))
+    adjusted_coords_xy = tf.nn.sigmoid(coords[:,:,:,0:2])
+    adjusted_coords_wh = tf.exp(tf.clip_by_value(coords[:,:,:,2:4], -1e3, 10)) * np.reshape(anchors, [1, 1, B, 2]) / np.reshape([W, H], [1, 1, 1, 2])
```

Comment:
what are the ranges of coords? why do we clip to, specifically, -1e3 and 10? Source?

Reply:
They were arbitrarily chosen. I had the problem where coords went to inf, so the main purpose is to clip the maximum. I'm actually not too sure how to choose the numbers.